packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1103,6 +1104,46 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// Override in subclasses with children and call the visitor for each child.
   void visitChildren(RenderObjectVisitor visitor) { }
 
+  /// Capture an image of the current state of the render object and its
+  /// children.
+  ///
+  /// The returned [ui.Image] has uncompressed raw bytes in the dimensions of
+  /// the render object, multiplied by the [pixelRatio].
+  ///
+  /// The property [isRepaintBoundary] must be true for [toImage] to work, and
+  /// the render object must not need painting (i.e. [debugNeedsPaint] must be
```

Comment:
rather than "must not need painting" it might be more positive to say "must have gone through the paint phase" or some such (the parenthetical is fine though)

Reply:
Done.